Corona-Warn-App/src/main/assets/default_stats/default_stats.bin

```diff
@@ -0,0 +1,3 @@
+
+
1
+
�ҟ�%https://corona-pandemieradar.de/:lang
```

Comment:
what's up with the weird characters before the url?

Reply:
it is a `bin` file when it is decoded it works fine